roles/common/tasks/config_validations.yml

```diff
@@ -158,3 +158,22 @@
     - fips_enabled | bool
     - ansible_os_family == "RedHat"
   tags: validate
+
+- assert:
+    that:
+      - oauth_enabled|bool
+    fail_msg: "Please enable OAuth on your cluster to setup LDAP with OAuth"
+  when:
+    - ldap_with_oauth_enabled | bool
+  tags: validate
+
+- assert:
+    that:
+      - "{{item}} != 'none'"
+    fail_msg: "Please provide the value of {{ item }} in the inventory file."
+  loop:
+    - oauth_token_uri
+    - oauth_issuer_url
```

Comment:
any specific reason for naming this one as _url vs above and below as _uri

Reply:
it is kept this way to stay aligned with the nomenclature we used for sso in C3